doc/rest.md

```diff
@@ -130,7 +130,11 @@ Body parameters:
       - JPEG (.jpg, .jpeg, with .jpg.aux.xml, .jgw, .jpgw, .jpegw or .wld)
       - any of above types in single ZIP file (.zip)
       - file names, i.e. array of strings
-   - more than one main raster file is allowed in combination with *time_regex* parameter
+   - publishing raster timeseries layers is allowed in case of:
+     - using *time_regex* parameter is mandatory
+     - more than one main file is allowed
+       - all of them are compressed in one zip file or all them are not compressed
+       - all main files have the same extension, compared in case-insensitive way
```

Comment:
- it is allowed to publish time-series layer by setting `time_regex` parameter and sending one or more main raster files (compressed in one archive or uncompressed) with the same extension